src/Engine/Combat/CombatEngine.java

```diff
@@ -20,7 +20,10 @@ public static void resolveCombat(Unit attacker, Unit defender, GameMap map)
     {
       modifiers.get(i).alterCombat(params);
     }
-    params.defender.damageHP(params.calculateDamage());
+    double damage = params.calculateDamage();
+    params.defender.damageHP(damage);
+    // TODO: balance this
+    params.attacker.CO.addStars(damage*params.defender.model.moneyCost/100000);
```

Comment:
Is this a DoR clone? I kind of like giving more stars to the one getting hammered; it makes sense from a "I'm getting wrecked, better activate rage mode" standpoint, and also from an anti-snowball standpoint.